engine/src/main/java/org/terasology/world/block/BlockRegion.java

```diff
@@ -76,15 +80,32 @@
         this.maxZ = maxZ;
     }
```

Comment:
This check is out of sync with `set(...)`. Can we just use 'set` in the constructor?

```java
    public BlockRegion(int minX, int minY, int minZ, int maxX, int maxY, int maxZ) {
      this.set(minX, minY, minZ, maxX, maxY, maxZ);
    } 
```